### PR TITLE
Reservation skeleton

### PR DESCRIPTION
**Pull request by eric-johnson**
SandersForPresident/BusRSVP#1

### CODE REVIEW

**Comment by maclover7** [comment]
Thanks for this PR! Can you add model tests + factories using Faker please?

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -4,4 +4,7 @@
   it 'has a valid factory' do
     expect(FactoryGirl.build(:user)).to be_valid
   end
+
```
Comment (maclover7) [inline spec/models/user_spec.rb]: Add test for `has_many :reservations`

File: spec/factories.rb [hunk 2]
```diff
@@ -2,6 +2,13 @@
 require 'factory_girl_rails'
 
 FactoryGirl.define do
+
+  factory :trip do
+    passenger_limit { Faker::Number.number(2) }
```
Comment (maclover7) [inline spec/factories.rb]: add name, description, host_id attrs to factory

File: spec/factories.rb [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
+  end
+
+  factory :reservation
```
Comment (maclover7) [inline spec/factories.rb]: add trip_id, passenger_id attrs to factory

**Comment by eric-johnson** [comment]
Feels like having your homework handed back by the teacher, but hey I learned something new about factories.

**Comment by ajwhite** [comment]
:clap: code reviews :tada: :smile:

**Comment by maclover7** [comment]
:+1: Thanks for this @eric-johnson!

**Pull request merged**